src/components/storage/storage.ts

```diff
@@ -0,0 +1,15 @@
+import { IProducts } from '../../types/product.interface';
+import Controller from '../controller/controller';
+
+class StorageData {
+  controller: Controller = new Controller();
+  createStateData(data: IProducts) {
+    const values = data?.products ? data.products : [];
+    localStorage.dataProducts = JSON.stringify(values);
+    console.log('put data products in localStorage');
+    localStorage.cartProducts?localStorage.cartProducts:localStorage.cartProducts=JSON.stringify([]);
+    this.controller.startAction();
+  }
+}
+
+export default StorageData;
```

Comment:
to be consistent choose one way of exporting for all class' exports

Reply:
set all class export default